unfoldingWord/checking-tool-wrapper
# Moved contextIdReducer, groupsDataReducer, groupsIndexReducer, commentsReducer, VerseEditsReducer, BookmarksReducer, SelectionsReducer to tools codebase

## Test instructions

- ### DO NOT MERGE UNTIL ALL PRs ARE READY TO BE MERGED 
- In translationCore repo checkout `feature-mannycolon-6501`
- run `rm -rf node_modules`  & `npm i`
- Open the app and test thoroughly  
- tN, Tw & wA tools should all work correctly




This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword/checking-tool-wrapper/79)
